apps/src/templates/VersionRow.jsx

```diff
@@ -67,7 +67,7 @@ export default class VersionRow extends React.Component {
       <tr className="versionRow">
         <td>
           <p>
-            Saved <UnlocalizedTimeAgo dateString={this.props.lastModified} />
+            <UnlocalizedTimeAgo dateString={this.props.lastModified} />
```

Comment:
is the removal of the string here intentional?

Reply:
hah, I should probably read the PR descriptions _before_ reviewing rather than _after_ :upside_down_face: 

I think it's probably still worth translating this, unless you've confirmed with starlabs that this change is otherwise desired

Reply:
I got sign-off from the Starlabs team for the new design. See the screenshot.
I just need to update some unit tests now.